src/cryptokit.ml

```diff
@@ -2072,6 +2074,98 @@ let new_key ?(rng = Random.secure_rng) ?e numbits =
 
 end
 
+module Paillier = struct
+
+  type key =
+    { size: int;
+      n: string;
+      n2: string;
+      g: string;
+      p: string;
+      q: string;
+      lambda: string;
+      mu: string}
+  let wipe_key k =
+    wipe_string k.n;
+    wipe_string k.n2;
+    wipe_string k.g;
+    wipe_string k.p;
+    wipe_string k.q;
+    wipe_string k.lambda;
+    wipe_string k.mu
+
+  let encrypt key msg ?r ()=
+    let check_random_number r =
+      if (Bn.(relative_prime r (Bn.of_bytes key.n)) && r<(Bn.of_bytes key.n)) then true else false in
```

Comment:
`if b then true else false` is just `b` :-)

Reply:
My bad